app/assets/stylesheets/application.css

```diff
@@ -13,3 +13,27 @@
  *= require_tree .
  *= require_self
  */
+
+ footer {
+   position: absolute;
+   bottom: 0;
+   left: 0;
+   width: 100%;
+   font-family: "futura-pt", Futura, "Helvetica Neue", "Helvetica", "Roboto", "Arial", sans-serif;
+   font-weight: 500;
+   letter-spacing: 0.05em;
+   text-rendering: optimizeLegibility;
```

Comment:
Can you let the people working on that know?

I'd probably put anything other than really super global styling (maybe lines 17-24) in a separate file, maybe `footer.css`.